frontend/src/components/WofIcon.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <div>
+        <i :class="icon" :style="`font-size: ${size}px`"></i>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'WofIcon',
+    props: {
+        icon: {
+            type: String,
+            required: true
+        },
+        size: {
+            type: Number,
+            default: 16
+        }
+    }
+}
+</script>
+
+<style lang="less">
```

Comment:
scoped or not?

Reply:
It doesn't matter here because this component will not have any other content.